dspace/config/emails/unsubscriptions

```diff
@@ -0,0 +1,24 @@
+# E-mail sent to designated address when unsubscribe eperson command was requested
+#
+# Parameters: {0} EPerson email
+#             {1} list of communities
+#             {2} list of collections
+#             {3} list of CRIS items
+#             {4} list of statistics
+#
+# See org.dspace.core.Email for information on the format of this file.
+#
+Subject: Unsubscribed eperson {0} from the following communities, collections, Cris items, statistics
```

Comment:
are you sure that the subject line is processed for placeholder? otherwise remove the {0} parameter and make a more general subject like
"Content and statistics subscription cancelled"

Reply:
Yes, it works.